website/docs/data-display/stacked-area-chart/examples/legend.tsx

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import {
+  Plot,
+  XAxis,
+  YAxis,
+  minMax,
+  StackedArea,
+  HoverLine,
+  makeDataHintsContainer,
+  LegendItem,
+  ChartLegend,
+} from '@semcore/ui/d3-chart';
+import { scaleLinear } from 'd3-scale';
+import { Flex, Box } from '@semcore/ui/flex-box';
+import { Text } from '@semcore/ui/typography';
+import { curveCardinal } from 'd3-shape';
+import resolveColor from '@semcore/utils/src/color';
+
+function formatDate(value, options) {
+  return new Intl.DateTimeFormat('en', options).format(value);
+}
+
+const lineColors = {
+  1: resolveColor('blue-300'),
+  2: resolveColor('green-200'),
+  3: resolveColor('orange-400'),
+};
+
+const dataHints = makeDataHintsContainer();
+
+export default () => {
+  const MARGIN = 40;
+  const width = 500;
+  const height = 300;
+
+  const xScale = scaleLinear()
+    .range([MARGIN, width - MARGIN])
+    .domain(minMax(data, 'time'));
+
+  const yScale = scaleLinear()
+    .range([height - MARGIN, MARGIN])
+    .domain([0, 15]);
+
+  const legendItems = Object.keys(data[0])
+    .filter((name) => name !== 'time')
+    .map((item) => {
+      return {
+        id: item,
+        label: `Stack ${item}`,
+        checked: true,
+        color: lineColors[item],
+      };
+    });
+
+  return (
+    <>
+      <ChartLegend.Flex dataHints={dataHints} items={legendItems} />
```

Comment:
Checkboxes look interactive but are not interactive actually. It feels weird

Reply:
added `Squrare` shape